templates/slurm.conf.j2

```diff
@@ -35,7 +35,7 @@ MpiDefault=none
 #PluginDir=
 #PlugStackConfig=
 #PrivateData=jobs
-ProctrackType=proctrack/pgid
+ProctrackType=proctrack/linuxproc # TODO: really want cgroup but needs cgroup.conf and workaround for CI
```

Comment:
hmm... I guess I don't understand the issues here.

Reply:
pgid is definitely the wrong choice (maybe it wasn't when the role was written). The correct choice is really cgroups but molecule tests run in containers where this won't work.